ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoHelpers.java

```diff
@@ -23,7 +25,7 @@
 
   public static final String METRO_HOST_PROP_NAME = "metro.host";
 
-  private static final int DEBUG_SERVER_HOST_PORT = 8081;
+  private static final int DEBUG_SERVER_HOST_PORT = ReactBuildConfig.DEBUG_SERVER_HOST_PORT;
   private static final int INSPECTOR_PROXY_PORT = 8081;
```

Comment:
@cpojer I noticed these ports are now the same on master. I wonder if this should now be:

```
private static final int DEBUG_SERVER_HOST_PORT = ReactBuildConfig.DEBUG_SERVER_HOST_PORT;
private static final int INSPECTOR_PROXY_PORT = ReactBuildConfig.DEBUG_SERVER_HOST_PORT;
```

Any idea?

Reply:
Can we use the same config by default but still allow it to be customized separately? I think that’s ideal.

Reply:
Yep. That makes sense to me.

Reply:
Confused myself during the rebase. This actually was removed and replaced with a resource value.

I can definitely do the same with the inspector proxy port. Just need to trouble shoot some issues running tests from a fresh setup.